spec/models/article_spec.rb

```diff
@@ -18,4 +18,19 @@
     expect(Article.first.desc_facet).to eq("desc_facet")
     expect(Article.first.geo_facet).to eq("geo_facet")
   end
+
+  it {should have_many(:tweets)}
+
+ xit "should build associated tweets from article url" do
+     article_data = [{
+      title: "title",
+      url: "http://example.com",
+      abstract: "abstract",
+      desc_facet: "desc_facet",
+      geo_facet: "geo_facet"
+    }]
+
+   article =  Article.create_articles(article_data)
```

Comment:
Useless assignment to variable - `article`. Did you mean `article_data`?<br>Unnecessary spacing detected.<br>Inconsistent indentation detected.